src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/IPCServicesTest.java

```diff
@@ -136,6 +142,13 @@ void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, InterruptedException.class);
     }
 
+    @Test
```

Comment:
This test does not work on Windows due to differences in how Windows works. This particular test should be skipped on Windows.

`@DisabledOnOs(OS.WINDOWS)`